boltzrpc/boltzrpc.proto

```diff
@@ -410,14 +410,16 @@ message CreateReverseSwapRequest {
     Pair pair = 4;
     repeated string chan_ids = 5;
     optional string wallet = 6;
+    // The daemon will return immediately after zero conf swaps aswell.
```

Comment:
Didn't always return instantly for 0-conf? Could you elaborate a little bit about the different behaviours here?

Reply:
Correct. And if this isnt set to true it will keep doing it, so its backwards compatible.
Clients who implement the update stream dont want that blocking call so they set it to true